integration_tests/__tests__/__snapshots__/generate_coverage_for_files.test.js.snap

```diff
@@ -0,0 +1,52 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`--generateCoverageForFiles 1`] = `
+"Test Suites: 2 passed, 2 total
+Tests:       2 passed, 2 total
+Snapshots:   0 total
+Time:        <<REPLACED>>
+Ran all test suites.
+"
+`;
+
+exports[`--generateCoverageForFiles 2`] = `
+" PASS  __tests__/b.test.js
+PASS  __tests__/a.test.js"
+`;
+
+exports[`--generateCoverageForFiles 3`] = `
+"----------|----------|----------|----------|----------|----------------|
+File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
+----------|----------|----------|----------|----------|----------------|
+All files |      100 |      100 |      100 |      100 |                |
+ a.js     |      100 |      100 |      100 |      100 |                |
+ b.js     |      100 |      100 |      100 |      100 |                |
+----------|----------|----------|----------|----------|----------------|
+"
+`;
+
+exports[`--generateCoverageForFiles 4`] = `
+"Test Suites: 1 passed, 1 total
+Tests:       1 passed, 1 total
+Snapshots:   0 total
+Time:        <<REPLACED>>
+Ran all test suites matching /a.js/i.
```

Comment:
this is incorrect (also broken in master for `--findRelatedTests`)
i filed the issue https://github.com/facebook/jest/issues/4247